src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,29 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
```

Comment:
Use destruction for `post`

Reply:
![image](https://github.com/mate-academy/react_static-list-of-posts-js/assets/143609932/784896c6-d083-4770-a08c-f62dd9bde0a9)
i did it